src/components/adminHome.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-const Home = () => {
+const AdminHome = () => {
 	return (
 		<div>
 			<h1> Hey I'm the Homepage </h1>
```

Comment:
Maybe change this to say "Admin Homepage" just so no one gets confused

Reply:
Good idea, should be resolved now